services/content-publishing/apps/worker/src/request_processor/dsnp.announcement.processor.ts

```diff
@@ -397,8 +400,11 @@ export class DsnpAnnouncementProcessor {
       icon: attachments,
       tag: this.prepareTags(content.profile.tag),
     };
+    const toUint8Array = new TextEncoder();
     const profileString = JSON.stringify(profileActivity);
-    const [cid, hash] = await this.pinBufferToIPFS(Buffer.from(profileString));
+    const profileEncoded = toUint8Array.encode(profileString);
+
+    const [cid, hash] = await this.pinBufferToIPFS(Buffer.from(profileEncoded));
```

Comment:
using text encoder to encode the note and profile strings that are pinned to ipfs